package.json

```diff
@@ -52,6 +53,7 @@
     "react-chartjs-2": "^2.7.4",
     "react-dom": "^16.5.0",
     "react-hot-loader": "^4.3.6",
+    "react-loadable": "^5.5.0",
```

Comment:
Where is this being used? Also, for lazy loading, you may want to look into `React.lazy` instead.